src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -401,24 +401,30 @@ class GrowableArrayWithAllocator : public GrowableArrayView<E> {
   }
 
   void push(const E& elem) { append(elem); }
-
-  E at_grow(int i, const E& fill = E()) {
+private:
+  static void default_fill(E* ptr) {
```

Comment:
This method is unused I believe

Reply:
I hoped I'd manage to push a fix for that before anyone saw it :-).